WooCommerce/Classes/ViewRelated/Dashboard/Free Trial/FreeTrialSurvey/FreeTrialSurveyViewModel.swift

```diff
@@ -0,0 +1,79 @@
+import SwiftUI
+
+/// View model for `FreeTrialSurveyView`
+///
+final class FreeTrialSurveyViewModel: ObservableObject {
+    @Published private(set) var selectedAnswer: SurveyAnswer?
+    @Published var otherReasonSpecified: String = ""
+
+    private let analytics: Analytics
+    private let source: FreeTrialSurveyCoordinator.Source
+
+    init(source: FreeTrialSurveyCoordinator.Source,
+         analytics: Analytics = ServiceLocator.analytics) {
+        self.source = source
+        self.analytics = analytics
+    }
+
+    var answers: [SurveyAnswer] {
+        SurveyAnswer.allCases
+    }
+
+    var feedbackSelected: Bool {
+        otherReasonSpecified.isNotEmpty || selectedAnswer != nil
+    }
+
+    func selectAnswer(_ answer: SurveyAnswer) {
+        selectedAnswer = answer
+    }
+
+    func submitFeedback() {
+        // TODO: 10266 Submit tracks
+    }
+
+    enum SurveyAnswer: String, Equatable, CaseIterable {
```

Comment:
Nit: `Equatable` can be inferred from the raw value type `String`, so it can be omitted.

Reply:
Removed in https://github.com/woocommerce/woocommerce-ios/pull/10319/commits/af1b1601909b73162dd3341b358ee84a05c1dbb4